src/Microsoft.NetCore.Analyzers/UnitTests/Security/UseXmlReaderForXPathDocumentTests.cs

```diff
@@ -113,14 +116,22 @@ public void TestMethod(XmlReader reader, XmlSpace space)
 }");
         }
 
-        protected override DiagnosticAnalyzer GetBasicDiagnosticAnalyzer()
+        private static async Task VerifyCSharp(string source, params DiagnosticResult[] expected)
         {
-            return new UseXmlReaderForXPathDocument();
-        }
+            var csharpTest = new VerifyCS.Test
+            {
+                TestCode = source,
+                TestBehaviors = TestBehaviors.SkipGeneratedCodeCheck
```

Comment:
📝 All the `VerifyCSharp` methods that only exist to set `TestBehaviors` like this can be converted to use `CSharpSecurityCodeFixVerifier` and remove the intermediate helper method.